File: geometry/area_formulas/python/area_formulas.py
import math

def triangle_area(base, height):
    return (1./2) * base * height

def square_area(edge_len):
    return edge_len**2
    
def rectangle(width, length):
    return length * height    
    
def ellipse(a, b):
    return math.pi * a * b

def trapezoid(base_one, base_two, height):
    return 0.5 * (base_one + base_two) * height
    
def circle(radius):
    return math.pi * (radius**2)

def cylinder(radius,height):
    return 2.0 * math.pi* radius * height;

def sphere(radius):
    return 4 * math.pi * (radius**2);

def sector(radius,angle):
    return radius*radius*angle*(1./2)

File: geometry/area_formulas/javascript/area_formulas.js
//
// ES6
const squareArea = (length) => {
  return Math.pow(length, 2);
}

const rectangleArea = (length, width) => {
  return length * width;
}

const triangleArea = (base, height) => {
  return (base*height) / 2;
}

const equilateralTriangleArea = (side) => {
  return Math.sqrt(3)/4 * Math.pow(side, 2);
}

const parallelogramArea = (base, height) => {
  return base*height;
}

const trapezoidArea = (base_one, base_two, height) => {
  return (base_one + base_two) * height / 2;
}

const circleArea = (radius) => {
  return Math.PI*(Math.pow(radius, 2));
}

const ellipseArea = (semi_major, semi_minor) => {
  return Math.PI*semi_major*semi_minor;
}
